api/metrics-api-util/src/main/java/org/hawkular/metrics/api/jaxrs/config/ConfigurationKey.java

```diff
@@ -40,7 +40,9 @@
             "CASSANDRA_MAX_REQUEST_CONN", false),
     WAIT_FOR_SERVICE("hawkular.metrics.waitForService", null, null, true),
     DEFAULT_TTL("hawkular.metrics.default-ttl", "7", "DEFAULT_TTL", false),
-    DISABLE_METRICS_JMX("hawkular.metrics.disable-metrics-jmx-reporting", null, "DISABLE_METRICS_JMX", true);
+    DISABLE_METRICS_JMX("hawkular.metrics.disable-metrics-jmx-reporting", null, "DISABLE_METRICS_JMX", true),
+    METRICS_PUBLISH_DELAY("hawkular-metrics.publish-delay", "1000", "METRICS_PUBLISH_DELAY", false),
+    METRICS_PUBLISH_PERIOD("hawkular-metrics.publish-period", "2000", "METRICS_PUBLISH_PERIOD", false);
```

Comment:
That's fine, although I don't think we should be guided by MIQ's impl for this :)  In general, people want alert as soon as possible.